packages/server/src/tools/dompurify.ts

```diff
@@ -0,0 +1,14 @@
+import { JSDOM } from "jsdom";
+import dompurify from "dompurify";
+
+interface ServerSideDomPurify {
+    sanitize: (unsafeHtmlInput: string) => string;
+}
+
+const serverSideDomPurify = () => {
+    const { window } = new JSDOM("<!DOCTYPE html>");
+    // @ts-ignore this is fine
+    return dompurify(window) as ServerSideDomPurify;
+};
+
+export default serverSideDomPurify();
```

Comment:
I ended up building a custom wrapper because in this context we'll only use `dompurify` in the server, and don't need all the other boilerplate. It was also a bit obfuscated, which isn't necessary.